plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtUnPlugNicCommandWrapper.java

```diff
@@ -82,4 +87,22 @@ public Answer execute(final UnPlugNicCommand command, final LibvirtComputingReso
             }
         }
     }
+
```

Comment:
could see same methods `getVlanIdFromBridgeName()` and `shouldDeleteBridge()` defined/implemented in the classes - LibvirtMigrateCommandWrapper, LibvirtStopCommandWrapper, LibvirtUnPlugNicCommandWrapper.

can you defined these in a common class (may be some util class) and use them across.